spec-0012/index.md

```diff
@@ -4,122 +4,180 @@ number: 12
+   For example:
+   - The expressions `--x` and `-~x` would be implicit subexpressions without spaces
+     containing more than one unary operator. The former can be simplified to `+x` or
+     simply `x`, and the latter requires explicit parentheses, i.e. `-(~x)`.
+   - The expression `x**y**z` would be an implicit subexpression without spaces
+     containing more than one `**` operator. This code would be executed as `x**(y**z)`
+     following the implicit order, but the explicit parentheses should be included for
+     clarity.
+   - In the expression `t**v*x**y + z`, no spaces are used around the multiplication
+     operator due to the presence of the lower-priority addition operator. However,
+     this would lead to `t**v*x**y` being an implicit subexpression without spaces
+     containing more than one `**` operator. This code would be executed as 
+    `(t**v)*(x**y) + z`, but the explicit parentheses should be included for clarity.  
+6. When breaking lines at operators, always include the operator at the beginning
+   of the new line rather than at the end of the old line, and use parentheses if
+   needed to allow for implicit rather than explicit line breaks. For example, if
+   `x + y` must be broken, prefer
+   ```python3
+   (x
+    + y)
+   ```
+   over
+   ```python3
+   x + \
+   y
+   ```
+7. If required to satisfy other style requirements, include line breaks between
+   the outermost explicit subexpression possible. For example, if
+   `t + (w + (x + (y + z))))` must be broken, prefer
+   ```python3
+   (t 
+    + (w + (x + (y + z)))))
+   ```
+   (if this break satisfies other requirements) over
+   ```python3
+   (t + (w + (x + (y 
+                   + z)))))
+   ```
+   If there are multiple candidates, include the break at the first opportunity.
+8. If line breaks must occur within a compound subexpression, the break should
+   be placed at the operator with lowest priority. For example, if
+   (x + y*z) must be broken, prefer
+   ```
+   (x
+    + y*z)
+   ```
+   over
+   ```
+   (x + y
+    * z)
+   ```
+   If there are multiple candidates, include the break at the first opportunity.
+9. All preceeding rules may be broken if there is a clear reason to do so. Examples include:
+   - They conflict with other style rules. For example, there is not supposed to be
+     whitepace surrounding the `**` operator, but one can imagine a chain of `**`
+     operations that exhausts the character limit of a line.
+  - Domain knowledge suggests a reason. For instance, in the expression
+    `t = (x + y) - z`, it may be important to emphasize that the addition should be
+    performed first for numerical reasons or because `(x + y)` is a conceptually
+    important quantity. In such cases, consider adding a comment, e.g.
+    ```python3
+    t = (x + y) - z  # perform `x + y` first for precision
+    ```
+    or breaking the expressions into separate logical lines, e.g.
+    ```
+    w = x + y
+    t = w - z
+    ```
```

Comment:
```suggestion
   - They conflict with other style rules. For example, there is not supposed to be
     whitepace surrounding the `**` operator, but one can imagine a chain of `**`
     operations that exhausts the character limit of a line.
   - Domain knowledge suggests a reason. For instance, in the expression
     `t = (x + y) - z`, it may be important to emphasize that the addition should be
     performed first for numerical reasons or because `(x + y)` is a conceptually
     important quantity. In such cases, consider adding a comment, e.g.
     ```python3
     t = (x + y) - z  # perform `x + y` first for precision
     ```
     or breaking the expressions into separate logical lines, e.g.
     ```python3
     w = x + y
     t = w - z
     ```
```